content/en/docs/armory-agent/armory-agent-quick.md

```diff
@@ -152,82 +145,115 @@ If you are not using Kustomize, you can still use the same manifests.
 
 ## Step 2: Agent installation
 
-### Kustomize
-
-Create the directory structure described below with `kustomization.yaml`, `kubesvc.yaml`, and `kubecfg/` containing the [kubeconfig files]({{< ref "manual-service-account" >}}) required to access target deployment clusters:
+Now lets deploy the Armory Agent itself. Armory provides a kustomize directory which already includes a basic configuration.
+You can download it by running these commands:
 
-```
-.
-├── kustomization.yaml
-├── kubesvc.yaml
-├── kubecfgs/
-│   ├── kubecfg-01.yaml
-│   ├── kubecfg-02.yaml
-│   ├── ...
-│   └── kubecfg-nn.yaml
+```bash
+mkdir armory-agent
+curl https://armory.jfrog.io/artifactory/manifests/kubesvc/armory-agent-{{<param kubesvc-version>}}-kustomize.tar.gz | tar -xJvf - -C armory-agent
 ```
 
-```yaml
-# ./kustomization.yaml
+You will have these files:
 
-# Namespace where you want to deploy the agent
+ * `kustomization.yaml` is kustomize entrypoint, refers and overrides the two other files.
+ * `kubesvc.yaml`  contains the [Agent options]({{< ref "agent-options" >}}):
+   * For installations without gRPC TLS connections, you should include `clouddriver.insecure: true` in the Agent options.
+   * For HA, make sure to set `clouddriver.grpc: clouddriver-ha-grpc-service.yaml:9091`
+ * `deployment.yaml` is the basic deployment maifest for Armory Agent
+
+If you already have a kustomization directory for your SpinnakerService manifest,
+you can include this `armory-agent` directory as a kustomize base:
+
+{{< prism lang="yaml" line="8" >}}
+# Existing kustomization.yaml
 namespace: spinnaker
+resources:
+  - spinnakerservice.yaml
+
 bases:
-  - https://armory.jfrog.io/artifactory/manifests/kubesvc/armory-agent-{{<param kubesvc-version>}}-kustomize.tar.gz
+  - agent-service
+  - armory-agent
 
-configMapGenerator:
-  - name: kubesvc-config
-    behavior: merge
-    files:
-      - kubesvc.yaml
+patchesStrategicMerge:
+  - agent-plugin/config.yaml
+  - agent-plugin/clouddriver-plugin.yaml
+{{</ prism >}}
 
-secretGenerator:
-  - name: kubeconfigs-secret
-    files:
-    # a list of all needed kubeconfigs
-    - kubecfgs/kubecfg-account01.yaml
-    - ...
-    - kubecfgs/kubecfg-account1000.yaml
-```
+### Configuring accounts
+
+Accounts are part of the [Agent Options]({{< ref "agent-options" >}}) which can be modified in `kubesvc.yaml` file
 
-`kubesvc.yaml`  contains the [Agent options]({{< ref "agent-options" >}}):
 ```yaml
 # ./kubesvc.yaml
-
 kubernetes:
   accounts:
   - name: account01
-    # /kubeconfigfiles/ is the path to the config files
-    # as mounted from the `kubeconfigs-secret` Kubernetes secret
     kubeconfigFile: /kubeconfigfiles/kubecfg-account01.yaml
-    ...
-  - ...
-...
 ```
 
-* For installations without gRPC TLS connections, you should include `clouddriver.insecure: true` in the Agent options.
```

Comment:
Was this info (and line below) moved somewhere else?

Reply:
yup, to [line 160](https://github.com/armory/docs/pull/671/files#diff-d51fcd92337deee3538163f79a916f3fe109ba4dd7ec6966019c375466dafbf3R160)